lib/features/opening_hours/domain/entities/timeslot.dart

```diff
@@ -2,17 +2,31 @@ import 'package:coffeecard/core/strings.dart';
 import 'package:equatable/equatable.dart';
 
 class Timeslot extends Equatable {
-  final int? start;
-  final int? end;
+  final (int, int)? start;
```

Comment:
Consider creating a typedef with named fields ("hours", "minutes") so it's more clear that this tuple represents a time of day